atd-vzv/src/views/dashboard/dashboard.js

```diff
@@ -3,8 +3,8 @@ import Fatalities from "./fatalities";
 import SeriousInjuries from "./seriousInjuries";
 import SeriousInjuryAndFatalCrashesByMonth from "./seriousInjuryAndFatalCrashesByMonth";
 import SeriousInjuryAndFatalCrashesByMode from "./seriousInjuryAndFatalCrashesByMode";
-import YearsOfLifeLost from "./yearsOfLifeLost";
 import FatalitiesMultiYear from "./FatalitiesMultiYear";
+import FatalitiesByTimeOfDayWeek from "./fatalitiesByTimeOfDayWeek";
```

Comment:
shouldn't we name these files with caps to be consistent with the component naming?

Reply:
Updated all component filenames!